Dockerfile

```diff
@@ -0,0 +1,48 @@
+#Another attempt at dockerizing frontend
+
+
+# Stage 1: Compile and Build angular codebase
+
+# Use official node image as the base image; Andrew's guide says to use node-14.17.3
```

Comment:
who's andrew, what guide?

Reply:
That comment was to remind me to tell my teammates this morning about the issue I had resolved and to give Andrew credit.  Now that I have passed on the info, I have removed the comment.